lib/Controller/ObjectsController.php

```diff
@@ -69,28 +69,49 @@ public function page(): TemplateResponse
     }
 
     /**
-     * Retrieves a list of all objects
-     *
-     * This method returns a JSON response containing an array of all objects in the system.
+     * Retrieves a list of all objects for a specific register and schema
      *
      * @NoAdminRequired
      * @NoCSRFRequired
      *
+     * @param string $register The register slug or identiefer
+     * @param string $schema The schema slug or identiefer
      * @return JSONResponse A JSON response containing the list of objects
      */
-    public function index(ObjectService $objectService, SearchService $searchService): JSONResponse
+    public function index(string $register, string $schema, ObjectService $objectService, SearchService $searchService): JSONResponse
```

Comment:
Is there a possbility register and schema are not given? Do we have a fallback error response if not?

Reply:
No, the path requires both parameters to be given